src/lib/components/Forbidden/Forbidden.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import Overridable from 'react-overridable';
+import { Link } from 'react-router-dom';
+import { Grid, Button, Icon } from 'semantic-ui-react';
+
+class Forbidden extends Component {
+  render() {
+    return (
+      <Overridable id="Forbidden.layout" {...this.props}>
+        <div className="frontsite">
+          <Grid
+            container
+            verticalAlign="middle"
+            textAlign="center"
+            className="not-found error-page"
```

Comment:
I am not able to found the declaration of the css classes `not-found`, `internal-server-error`, and `unauthorized`. If they are not used, I would suggest removing them, wdyt?

Reply:
This was meant for custom styling, instead of override the entire component (similar to what was done in `NotFound`). But I agree that `error-page` is more than enough for most purposes - and overriding is still possible in case of further modifications.